test/functional/apps/visualize/_point_series_options.js

```diff
@@ -203,11 +204,87 @@ export default function ({ getService, getPageObjects }) {
 
         const labels = await PageObjects.visualize.getXAxisLabels();
 
-        await kibanaServer.uiSettings.replace({ 'dateFormat:tz': 'Browser' });
+        expect(labels).to.eql(expectedLabels);
+      });
+
+      it('should show different labels in different timezone', async function () {
+        const fromTime = '2015-09-22 09:05:47.415';
+        const toTime = '2015-09-22 16:08:34.554';
+        // note that we're setting the absolute time range while we're in 'America/Phoenix' tz
+        await PageObjects.timePicker.setAbsoluteRange(fromTime, toTime);
+        await PageObjects.common.sleep(10000);
```

Comment:
Waiting 10 seconds seems a bit excessive - is `waitForVisualizationRenderingStabilized` not stable enough here?

Reply:
agreed, changes coming in next push.